ricecooker/utils/pipeline/transfer.py

```
@@ -237,6 +238,131 @@ def handle_file(self, path, yt_dlp_settings=None):
+            # Default to PDF for unknown Google Workspace types
+            export_type = "application/pdf"
+        return export_type
+
+    def handle_file(self, path: str):
+        try:
+            from googleapiclient.http import MediaIoBaseDownload
+        except ImportError:
+            raise RuntimeError(
+                "Google Drive downloads require google-api-python-client library\n"
+                + instructions
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)